DATA/FX/MISC/misc_ale.ini

```diff
@@ -2608,3 +2608,74 @@ textures = fx\miningship.txm
 textures = fx\smoke.txm
 textures = fx\animated.txm
 
+[VisEffect]
+nickname = rtc_gas_pocket

```

Comment:
"rtc" is the wrong pattern for those effects. RTC is a shortcut for something related to filmographic cutscenes.